gnovm/stdlibs/std/crypto.go

```diff
@@ -8,3 +8,7 @@ func X_derivePkgAddr(pkgPath string) string {
 	addr := string(gno.DerivePkgBech32Addr(pkgPath))
 	return addr
 }
+
+func X_deriveStorageDepositAddr(pkgPath string) string {
```

Comment:
What’s the expected usage of this stdlib addition?

Reply:
The storage deposit is saved in a separate address account. It is different from the realm address account used for receiving tokens.

It is used to query the storage deposit amount of a realm and to tally the total.